Description: Detection for Mac OS X's launchd service on Mac OS X 10.10 and beyond
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/nxcomp/Loop.cpp
+++ b/nxcomp/Loop.cpp
@@ -4183,7 +4183,7 @@
 
   #ifdef __APPLE__
 
-  if (strncasecmp(display, "/tmp/launch", 11) == 0)
+  if ((strncasecmp(display, "/tmp/launch", 11) == 0) || (strncasecmp(display, "/private/tmp/com.apple.launchd", 30) == 0))
   {
     #ifdef TEST
     *logofs << "Loop: Using launchd service on socket '"
